app/templates/datasets.hbs

```diff
@@ -54,14 +54,11 @@
               .csv
             </a>
 
-            {{#if model.raw_data.spatialMetaData}}
+            {{#unless model.raw_data.spatialMetaData}}
               <a class="button lift" href={{download_link_shapefile}} download={{model.dataset.table_name}}>
                 .shp
               </a>
-              <a class="button lift" href={{download_link_geojson}} download={{model.dataset.table_name}}>
-                .geojson
-              </a>
```

Comment:
This PR removes GeoJSON support?

Reply:
Not specifically but GeoJSON is dependent upon CartoDB so we would need to build GeoJSON endpoints as a new Data common feature when we pull out Carto. I can leave it there but at some point we should remove it from the UI until it is rebuilt.